### PR TITLE
git-show-tree: add page

### PR DESCRIPTION
**Pull request by CleanMachine1**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

For #5137

### CODE REVIEW

**Review by mfrw — COMMENTED** [review]
I rephrased the example .. thoughts ?

File: pages/common/git-show-tree.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# git show-tree
+
+> Show a decorated tree graph for each branch, showing commits other statistics.
```
[the remark this answers is not in the document]
Comment (CleanMachine1) [reply pages/common/git-show-tree.md:3]: `git log --all --graph --decorate --oneline --simplify-by-decoration` ( from the repo here https://github.com/tj/git-extras/blob/master/bin/git-show-tree )

[the remark this answers is not in the document]
Comment (CleanMachine1) [reply pages/common/git-show-tree.md:3]: Also what about the following 

```suggestion
> Show a decorated tree graph for a Git repository, showing annotations.
```

[the remark this answers is not in the document]
Comment (mfrw) [reply pages/common/git-show-tree.md:3]: > `git log --all --graph --decorate --oneline --simplify-by-decoration` ( from the repo here https://github.com/tj/git-extras/blob/master/bin/git-show-tree )

wohoooo .. i was pretty close :)

[the remark this answers is not in the document]
Comment (marchersimon) [reply pages/common/git-show-tree.md:3]: I like the last one, but it's not obvious what kind of tree it is showing. How about:
```suggestion
> Show a decorated tree graph with all branches of a Git repository, showing annotations.
```

**Review by marchersimon — APPROVED** [review]
👍🏽

**Review by SethFalco — APPROVED** [review]

**Review by sbrl — APPROVED** [review]
Thanks, @CleanMachine1!

**Pull request merged**